optuna/integration/mxnet.py

```diff
@@ -0,0 +1,72 @@
+from __future__ import absolute_import
+
+import optuna
+
+try:
+    import mxnet as mx  # NOQA
+    _available = True
+except ImportError as e:
+    _import_error = e
+    _available = False
+
+
+class MxnetPruningCallback(object):
+    """MXNet callback to prune unpromising trials.
+
+    Example:
+
+        Add a pruning callback which observes validation accuracy.
+
+        .. code::
+
+            model.fit(train_data=X, eval_data=Y,
+                      eval_end_callback=MxnetPruningCallback(trial, eval_metric='accuracy'))
+
+    Args:
+        trial:
+            A :class:`~optuna.trial.Trial` corresponding to the current evaluation of the
+            objective function.
+        eval_metric:
+            An evaluation metric name for pruning, e.g., ``cross-entropy`` and
+            ``accuracy``. If using default metrics like mxnet.metrics.Accuracy, use it's
+            default metric name. For custom metrics, use the metric_name provided to
+             constructor. Please refer to `mxnet.metrics reference
```

Comment:
The generated documentation has an unintended output (see below)  because of the extra whitespace before "constructor".
![image](https://user-images.githubusercontent.com/181413/56564214-88982080-65e8-11e9-8b15-29c77e58816c.png)

```suggestion
            constructor. Please refer to `mxnet.metrics reference
```